client/web/src/enterprise/user/settings/auth/UserSettingsPermissionsPage.tsx

```diff
@@ -17,13 +39,44 @@ import styles from './UserSettingsPermissionsPage.module.scss'
  */
 export const UserSettingsPermissionsPage: React.FunctionComponent<
     React.PropsWithChildren<{
-        user: UserSettingsAreaUserFields
+        user: { id: string; username: string }
     }>
 > = ({ user }) => {
-    useEffect(() => eventLogger.logViewEvent('UserSettingsPermissions'))
-    const permissionsInfo = useObservable(useMemo(() => userPermissionsInfo(user.id), [user.id]))
+    useEffect(() => eventLogger.logViewEvent('UserSettingsPermissions'), [])
+
+    const [{ q }, setSearchQuery] = useURLSyncedState({ q: '' })
+    const debouncedQ = useDebounce(q, 300)
```

Comment:
full var names fan here 👋🏻 
```suggestion
    const debouncedQuery = useDebounce(query, 300)
```

Reply:
updated